python/torch_mlir/dialects/torch/__init__.py

```diff
@@ -4,4 +4,4 @@
 # Also available under a BSD-style license. See LICENSE.
 
 from .._torch_ops_gen import *
-from ..._mlir_libs._torchMlir import register_dialect
```

Comment:
With the new scheme, do you think we need this at all? Iiuc, it was mainly here because there was no other way to register things (and now it is automatic).